### PR TITLE
ButterCMS Example

### PR DESCRIPTION
**Pull request by chibicode**
[ch2580]

**Demo:** https://next-blog-buttercms.now.sh/
**Preview Mode:** https://next-blog-buttercms.now.sh/api/preview?secret=CdEAKMRT4tAwzqiD&slug=draft-post-test

### CODE REVIEW

File: examples/cms-buttercms/.gitignore [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (Timer) [inline examples/cms-buttercms/.gitignore:1]: This file shouldn't be necessary.

File: examples/cms-buttercms/.env.local.example [hunk 2]
```diff
@@ -0,0 +1,2 @@
+NEXT_EXAMPLE_CMS_BUTTERCMS_API_KEY=
```
Comment (Timer) [inline examples/cms-buttercms/.env.local.example:1]: Missing comment to explain to rename to `.env.local`.

Comment (chibicode) [reply examples/cms-buttercms/.env.local.example:2]: Done!

File: examples/cms-buttercms/.gitignore (hunk 1 above)
Comment (chibicode) [reply examples/cms-buttercms/.gitignore]: Removed!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.7s | ⚠️ +413ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.808 | 1.825 | ⚠️ +0.02 |
| / avg req/sec | 1382.46 | 1369.6 | -12.86 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.191 | 1.225 | ⚠️ +0.03 |
| /error-in-render avg req/sec | 2098.36 | 2040.27 | -58.09 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| index.html gzip | 901 B | 901 B | ✓ |
| link.html gzip | 907 B | 907 B | ✓ |
| withRouter.html gzip | 894 B | 894 B | ✓ |
| Overall change | 2.7 kB | 2.7 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 11.3s | 11.5s | ⚠️ +156ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _error.js | 874 kB | 874 kB | ✓ |
| 404.html | 4.13 kB | 4.13 kB | ✓ |
| hooks.html | 3.71 kB | 3.71 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 907 kB | 907 kB | ✓ |
| withRouter.js | 907 kB | 907 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 12.7s | 13.1s | ⚠️ +414ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.164 | 2.103 | -0.06 |
| / avg req/sec | 1155.26 | 1188.65 | ⚠️ +33.39 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.399 | 1.437 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 1786.56 | 1739.9 | -46.66 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| index.html gzip | 901 B | 901 B | ✓ |
| link.html gzip | 907 B | 907 B | ✓ |
| withRouter.html gzip | 894 B | 894 B | ✓ |
| Overall change | 2.7 kB | 2.7 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 13.9s | 13.7s | -162ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _error.js | 874 kB | 874 kB | ✓ |
| 404.html | 4.13 kB | 4.13 kB | ✓ |
| hooks.html | 3.71 kB | 3.71 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 907 kB | 907 kB | ✓ |
| withRouter.js | 907 kB | 907 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.3s | -141ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.244 | 2.136 | -0.11 |
| / avg req/sec | 1113.95 | 1170.25 | ⚠️ +56.3 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.451 | 1.372 | -0.08 |
| /error-in-render avg req/sec | 1723.18 | 1821.66 | ⚠️ +98.48 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| index.html gzip | 895 B | 895 B | ✓ |
| link.html gzip | 901 B | 901 B | ✓ |
| withRouter.html gzip | 888 B | 888 B | ✓ |
| Overall change | 2.68 kB | 2.68 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| buildDuration | 13.1s | 13.1s | -52ms |
| nodeModulesSize | 64.9 MB | 64.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.49 kB | 6.49 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4b11.js gzip | 10.9 kB | 10.9 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 7.26 kB | 7.26 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error.js gzip | 3.25 kB | 3.25 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 8.22 kB | 8.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| _error.module.js gzip | 2.04 kB | 2.04 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.33 kB | 5.33 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _buildManifest.js gzip | 171 B | 171 B | ✓ |
| _buildManife..dule.js gzip | 180 B | 180 B | ✓ |
| Overall change | 351 B | 351 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | chibicode/next.js cms-buttercms | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.13 kB | 4.13 kB | ✓ |
| hooks.html | 3.71 kB | 3.71 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 907 kB | 907 kB | ✓ |
| withRouter.js | 907 kB | 907 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>

**Review by Timer — APPROVED** [review]
LGTM!

**Pull request merged**